src/render/OpenGL.cpp

```diff
@@ -59,6 +59,23 @@ CHyprOpenGLImpl::CHyprOpenGLImpl() {
     m_tGlobalTimer.reset();
 }
 
+void CHyprOpenGLImpl::logError(const GLuint& shader, bool program) {
```

Comment:
`logShaderError`, `logError` is misleading

Reply:
Should also be good